Review of empirical-process-control (#3)
lapaevrv/translation: principles/empirical-process-control.md

---

> title: Эмпирический Контроль Процесса

Reviewer: ```suggestion
title: Процесс эмпирического управления
```

> title: Эмпирический Контроль Процесса

Reviewer: ```suggestion
title: Empirical Process Control
```

order: 50
---

*Empirical process control* is a core Scrum principle, and distinguishes it from other agile frameworks. The *Scrum Guide* puts it well:

> > Scrum is not a process or a technique for building products; rather, it is a framework within which you can employ various processes and techniques. Scrum makes clear the relative efficacy of your product management and development practices so that you can improve.

Reviewer: ```suggestion
> Scrum is not a process, technique, or definitive method. Rather, it is a framework within which you can employ various processes and techniques. Scrum makes clear the relative efficacy of your product management and work techniques so that you can continuously improve the product, the team, and the working environment.
```

Reviewer: Пока решили оставить как в оригинале, соотвественно в переводе тоже берём как на сайте scrumguides.
